src/components/UpdateProfile/UpdateProfileStyled.ts

```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components';
+import * as colors from 'styles/variables';
+
+export const WrapStyled = styled.div.attrs(() => ({
+  className: 'msd__user-profile__wrapper'
+}))`
+  * {
```

Comment:
i think you can remove the whole asterisk section. it assigns the same styles as below anyway, and its scope is everything within this styled component, so it's not needed